chain/src/chain.rs

```diff
@@ -278,18 +278,20 @@ impl ChainService {
     // process_block will do block verify
     // but invoker should guarantee block header be verified
     pub fn process_block(&mut self, block: Arc<BlockView>, switch: Switch) -> Result<bool, Error> {
-        debug!("begin processing block: {}", block.header().hash());
+        let block_number = block.number();
+        let block_hash = block.hash();
+
+        debug!("begin processing block: {}-{}", block_number, block_hash);
+        let timer = Timer::start();
         if block.header().number() < 1 {
-            warn!(
-                "receive 0 number block: {}-{}",
-                block.header().number(),
-                block.header().hash()
-            );
+            warn!("receive 0 number block: 0-{}", block_hash);
         }
-        self.insert_block(block, switch).map(|ret| {
-            debug!("finish processing block");
-            ret
-        })
+        let is_better = self.insert_block(block, switch)?;
```

Comment:
`is_better` seems used for checking the block is tip or uncle block, so I think `is_tip` or `is_best` will be more meaningful